src/instance.ts

```diff
@@ -732,6 +733,57 @@ class Instance extends common.GrpcServiceObject {
     );
   }
 
+  /**
+   * Get the instance's end point Uris.
+   *
+   * Wrapper around {@link v1.InstanceAdminClient#getInstance}.
+   *
+   * @see {@link v1.InstanceAdminClient#getInstance}
+   * @see [GetInstance API Documentation](https://cloud.google.com/spanner/docs/reference/rpc/google.spanner.admin.instance.v1#google.spanner.admin.instance.v1.InstanceAdmin.GetInstance)
+   *
+   * @param {GetInstanceEndPointUrisCallback} [callback] Callback function.
+   * @returns {Promise<string[]>}
+   *
+   * @example
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const spanner = new Spanner();
+   *
+   * const instance = spanner.instance('my-instance');
+   *
+   * instance.getInstanceEndPointUris(function(err, endPointUris) {});
+   *
+   * //-
+   * // If the callback is omitted, we'll return a Promise.
+   * //-
+   * instance.getInstanceEndPointUris().then(function(data) {
+   *   const endPointUris = data[0];
+   * });
+   */
+  getInstanceEndPointUris(
+    callback?: GetInstanceEndPointUrisCallback
+  ): Promise<string[]> | void {
+    const reqOpts = {
+      name: this.formattedName_,
+      fieldMask: {
+        paths: ['endpointUris'],
+      },
+    };
+    this.request<Instance>(
+      {
+        client: 'InstanceAdminClient',
+        method: 'getInstance',
+        reqOpts,
+      },
+      (err, resp) => {
+        if (err) {
+          callback!(err);
+          return;
+        }
+        callback!(null, resp!.endpointUris);
+      }
+    );
+  }
+
```

Comment:
probably this method will be removed and will use existing method `getMetadata`, once @AVaksman expose the configuration part in that method.